pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -209,6 +209,64 @@ func TestPriorityQueue_AddUnschedulableIfNotPresent(t *testing.T) {
 	}
 }
 
+func TestPriorityQueue_AddUnschedulableIfNotPresent_Async(t *testing.T) {
```

Comment:
I'd like to extend this case a little bit. Right now, this covers the case that "Pod i" is in a goroutine, and it's ensured to be put back into activeQ properly. But the case could be that **"Pod i to j"** are in goroutines, and **they** should be put back into activeQ. This is the case my previous PR didn't cover.

I drafted a new testcase FYI:

https://github.com/Huang-Wei/kubernetes/blob/46fe8e566b7b9435fe905e5e832fff60b02106cb/pkg/scheduler/internal/queue/scheduling_queue_test.go#L212-L264

Reply:
Thanks! Your test is much better than mine. Updated.

Reply:
Thanks, @cofyc and @Huang-Wei.
Please add a comment above the test and describe the test scenario.